src/psyclone/psyad/lfric/Makefile

```diff
@@ -0,0 +1,54 @@
+TL_DIR = tangent_linear
+TL_TWEAKED_DIR = tangent_linear_tweaked
+ADJ_PARTIAL_DIR = adjoint_partial
+
+all: ${ADJ_PARTIAL_DIR}/adj_dg_inc_matrix_vector_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_moist_dyn_mass_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_project_eos_pressure_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_hydrostatic_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_transpose_matrix_vector_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_kinetic_energy_gradient_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_vorticity_advection_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_moist_dyn_gas_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_pressure_gradient_bd_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_matrix_vector_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_rhs_project_eos_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_rhs_sample_eos_kernel_mod.F90
+
+# Kernels that psyad can process
+
+${ADJ_PARTIAL_DIR}/adj_rhs_project_eos_kernel_mod.F90: ${TL_DIR}/tl_rhs_project_eos_kernel_mod.F90
```

Comment:
Moved the examples, however I've not made use of the harness for the time being as I think that is a separate PR, given that not all files work with the harness (the `tl_pressure_gradient_bd_kernel_mod.F90` kernel fails as it has a stencil) and that it would make sense to put the generated files in a different location before compiling.

Reply:
OK. I think having PSyAD run at least is a big step forward.